letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
unfortunately, it doesn't. With storage per database (in the original version, you have one key/certificate per database, because the letsencrypt-folder is a sibling of the filestore folder), we can separate requesting certificates for different scenarios on the same instance. This won't work with this implementation proposal. For any SaaS-scenario, it's also completely unviable.

What you could do before is

```
server {
    server_name your_domain;
    location /.well-known {
        proxy_set_header X-Odoo-dbfilter ^your_database_with_letsencrypt_installed$;
    }
```

in nginx (adapt for other proxies) to fix your problem, provided dbfilter_from_header is installed, and I'd rather use that for supporting multiple databases on the same domain. As soon as you get serious with the website module, you'll need one domain per database anyways, so yes, I consider everything else than a dbfilter to point to the right database to use a hack.